pkg/kubelet/prober/prober.go

```diff
@@ -231,11 +231,7 @@ func (p *prober) newExecInContainer(container api.Container, containerID kubecon
 		var buffer bytes.Buffer
 		output := ioutils.WriteCloserWrapper(&buffer)
 		err := p.runner.ExecInContainer(containerID, cmd, nil, output, output, false, nil)
-		if err != nil {
-			return nil, err
-		}
-
-		return buffer.Bytes(), nil
+		return buffer.Bytes(), err
```

Comment:
@ingvagabund line 2887 probably needs to change as well. 

@sttts I agree that this isn't an ideal fix, but it does effectively restore the code to the same unidiomatic way it was before... I can work more on it this afternoon, or I can put in a TODO to fix this properly later.